cloud-controller-manager/osc/instances_v2.go

```diff
@@ -245,6 +245,11 @@ func parseInstanceIDFromProviderIDV2(providerID string) (string, error) {
 	// * /<availability-zone>/<instance-id>
 	// * <instance-id>
```

Comment:
If we want to still keep the possibility to set only the instance Id, check if the instance-id is `i-*`